Resources/Syncfusion.SfScheduler.WPF.fr-FR.resx

```diff
@@ -242,35 +242,35 @@
     <comment>Text for Hour</comment>
   </data>
   <data name="HoursIntervalTypeReminder" xml:space="preserve">
-    <value>Heures</value>
+    <value>Intervalle d'heures pour le rappel</value>
     <comment>Text for Hours</comment>
   </data>
   <data name="InValidEndTimeMessageAlert" xml:space="preserve">
-    <value>La date de fin que vous avez entrée se produit avant la date de début.</value>
+    <value>Alerte de message de fin non valide</value>
     <comment>Message alert for invalid end time</comment>
   </data>
   <data name="InvalidOccurrenceEditingMessageAlert" xml:space="preserve">
-    <value>Deux occurrences de [] ne peuvent pas se produire le même jour.</value>
+    <value>Alerte de message d'édition d'occurrence non valide</value>
     <comment>'[]' replaced by selected appointemnt subject</comment>
   </data>
   <data name="January" xml:space="preserve">
-    <value>janvier</value>
+    <value>Janvier</value>
     <comment>Text for January</comment>
   </data>
   <data name="July" xml:space="preserve">
-    <value>juillet</value>
+    <value>Juillet</value>
     <comment>Text for July</comment>
   </data>
   <data name="June" xml:space="preserve">
-    <value>juin</value>
+    <value>Juin</value>
     <comment>Text for June</comment>
   </data>
   <data name="JustThisOne" xml:space="preserve">
-    <value>Juste celui-là</value>
+    <value>Seulement celle-ci</value>
```

Comment:
Check this and confirm

Reply:
Checked, getting same text
![image](https://github.com/user-attachments/assets/ffaf4bae-5b04-4c40-a7fa-3b0cbd1b5493)